kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/utils/OffsetFinder.java

```diff
@@ -89,11 +89,11 @@ public void findEntryComplete(Position position, Object ctx) {
         checkArgument(ctx instanceof AsyncCallbacks.FindEntryCallback);
         AsyncCallbacks.FindEntryCallback callback = (AsyncCallbacks.FindEntryCallback) ctx;
         if (position != null) {
-            log.info("[{}][{}] Found position {} closest to provided timestamp {}", position,
+            log.info("Found position {} closest to provided timestamp {}", position,
```

Comment:
also here, log the information of managedLedger.getName() and timestamp is useful